prime-router/metadata/HL7/catchall/hl7/resource/Organization.yml

```diff
@@ -1,202 +0,0 @@
-# $schema: ./../../../../json_schema/fhir/hl7-to-fhir-mapping-resource-template.json
-
-# Used by most organizations
-# $orgIdValue would be missing because not passed in such as for IN1 / IN2 
-resourceType: Organization
-id_1:
-  condition: $orgIdValue NULL
-  type: STRING
-  valueOf: "GeneralUtils.generateResourceId()"
-  expressionType: JEXL
-
-# When there is IN1 / IN2 record but there is NO TENANT
-# $orgIdValue must be passed in to control which IN1 / IN2 is used, and prevent bleed
-id_2:
-  condition: $orgIdValue NOT_NULL && $TENANT NULL
-  type: VALID_ID
-  valueOf: $orgIdValue
-
-# When there is IN1 / IN2 record and there IS a TENANT
-# $orgIdValue must be passed in to control which IN1 / IN2 is used, and prevent bleed
-id_3:
-  condition: $orgIdValue NOT_NULL && $TENANT NOT_NULL
-  type: VALID_ID
-  valueOf: $orgIdWithTenantPrefix
-  vars:
-    orgIdWithTenantPrefix: $TENANT + $period + $orgIdValue
-  constants:
-    period: "." # period is constant used between concatention in $orgIdWithTenantPrefix
-
-# Used by most organizations
-identifier_1:
-  condition: $id NOT_NULL
-  valueOf: datatype/Identifier_Gen
```

Comment:
When I ran the data-mapping tool it said that `Identifier_XON` and `Identifier_Gen` are not referenced.

Reply:
good catch